### DISCUSSION TITLE
RCS Transient

### DISCUSSION THREAD

**Discussion by MBlan18** [MikaylaFischler/cc-mek-scada#495] [Support] [answered]
Just wondering what this is and how I can resolve it so that it stops flashing lights on my monitors. Something about Boil Rate mismatch?

**Comment by MikaylaFischler** [comment, marked as answer]
[RCS transient](https://github.com/MikaylaFischler/cc-mek-scada/wiki/Alarms#rcs-transient-conditions) means something is wrong with your cooling system, such as a [boil rate mismatch](https://github.com/MikaylaFischler/cc-mek-scada/wiki/Annunciator-Panels#rcs-panel). Boil rate mismatch means your boiler (or reactor) is making a different amount of steam than the turbine(s) is/are receiving, which could mean a few things, such as not having enough flow through pipes. If that isn't the case (such as no pipes), you may have a misconfigured setup, such as more than one reactor per turbine or your turbines not connected to an RTU.
